man/CMakeLists.txt

```diff
@@ -13,45 +13,55 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1335 USA
 
-SET(MAN1_WSREP wsrep_sst_rsync.1 wsrep_sst_common.1 wsrep_sst_mariabackup.1
-    wsrep_sst_mysqldump.1 wsrep_sst_rsync_wan.1 galera_recovery.1 galera_new_cluster.1)
-SET(MAN1_SERVER innochecksum.1 myisam_ftdump.1 myisamchk.1
-                aria_chk.1 aria_dump_log.1 aria_ftdump.1 aria_pack.1 aria_read_log.1
-                aria_s3_copy.1
-		myisamlog.1 myisampack.1 mysql.server.1 mariadb-conv.1
-		mysql_fix_extensions.1
-		mysql_install_db.1
-		mysql_upgrade.1
-		mysqld_multi.1 mysqld_safe.1
-		resolveip.1 mariadb-service-convert.1
-		mysqld_safe_helper.1
-		mysql_ldb.1 myrocks_hotbackup.1
-		mbstream.1 mariabackup.1)
 IF(WITH_WSREP)
-  SET(MAN1_SERVER ${MAN1_SERVER} ${MAN1_WSREP})
+  SET(MAN1_WSREP wsrep_sst_rsync.1 wsrep_sst_common.1 wsrep_sst_mariabackup.1
+      wsrep_sst_mysqldump.1 wsrep_sst_rsync_wan.1 galera_recovery.1 galera_new_cluster.1)
 ENDIF()
+SET(MAN1_SERVER_COMPAT mysql.server.1
+                       mysql_fix_extensions.1
+                       mysql_install_db.1
+                       mysql_upgrade.1
+                       mysqld_multi.1 mysqld_safe.1
+                       mysqld_safe_helper.1)
+SET(MAN1_SERVER_NATIVE innochecksum.1 myisam_ftdump.1 myisamchk.1
+                       aria_chk.1 aria_dump_log.1 aria_ftdump.1 aria_pack.1 aria_read_log.1
+                       myisamlog.1 myisampack.1  mariadb-conv.1
+                       resolveip.1 mariadb-service-convert.1)
+SET(MAN1_SERVER ${MAN1_SERVER_COMPAT} ${MAN1_SERVER_NATIVE})
 SET(MAN8_SERVER mysqld.8)
```

Comment:
no compat/native for MAN8? what about mariadbd.8?

Reply:
its created by the `MARIADB_SYMLINK_MANPAGE` macro and placed in the `ManPagesServer` component.

Reply:
that won't do. `mariadbd.8` is just
```troff
.so man8/mysqld.8
```
while it should be the other way around